package.json

```diff
@@ -33,7 +33,11 @@
   },
   "homepage": "https://github.com/fastpack/fastpack#readme",
   "resolutions": {
-    "@esy-ocaml/reason": "github:facebook/reason#335c3fb2470964c2b4a7d571a8fbd9acdc3c2323"
+    "ocaml": "github:esy-ocaml/ocaml#1933caa",
```

Comment:
This `ocaml` override is temporary until this change is picked up: https://github.com/esy-ocaml/ocaml/pull/9